README.md

```diff
@@ -13,16 +13,5 @@ pip install geojson2osm
 ## Usage
 
 ```python
-import json
```

Comment:
Lets include both , the import  as a python lib and from the command line

Reply:
Done.